src/static/styles/UploadDocs.scss

```diff
@@ -0,0 +1,12 @@
+
+.card-alignment {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%; // make sure the parent is full screen 
```

Comment:
have you tried this on mobile?

Reply:
No unfortunately I didn't have the time :(